docs/oauth2-examples-multiresource.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Use multiple OAuth 2.0 servers and/or audiences
+displayed_sidebar: docsSidebar
+---
+<!--
+Copyright (c) 2005-2024 Broadcom. All Rights Reserved. The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+
+All rights reserved. This program and the accompanying materials
+are made available under the terms of the under the Apache License,
+Version 2.0 (the "License”); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Use multiple OAuth 2.0 servers and/or audiences
+
+Demonstrate how to authenticate using OAuth 2.0 protocol
+and several OAuth resources using the following flows:
+
+* Access AMQP protocol
+* Access Management UI
+
+## Prerequisites to follow this guide
+
+- Docker
+
+## Motivation
+
+All the examples demonstrated by this tutorial, except for this use case, configure a single **resource_server_id** and therefore a single **OAuth 2.0 server**.
+
+You could encounter scenarios where some management users and/or applications are registered in
+different OAuth2 servers or they could be registered on the same OAuth2 server however they could refer to RabbitMQ with different audience values. When this happens, you have to declare as many OAuth2 resources as audiences and/or authorization servers.
+
+The following section demonstrates a scenario where users are declared in **Keycloak** however they refer to RabbitMQ with two distinct **audience**, one `rabbit_prod` and the other `rabbit_dev`.
+
+## AMQP clients and management users registered in same OAuth 2.0 server but with different audience
+
+RabbitMQ is configured with two OAuth2 resources one called `rabbit_prod` and another `rabbit_dev`. Think that the
+production team refers to RabbitMQ with the `rabbit_prod` audience. And the development team with the `rabbit_dev` audience.
+Because both teams are registered in the same OAuth2 server you are going to configure its settings such as `jwks_url` at the
+root level so that both resources share the same configuration.
+
```

Comment:
The above is really well explained ...Instead of "Think that the" how about saying "For example purposes, let's say....." 

GLOBAL COMMENT: Use of "demonstrating/demonstration" to me a demonstration is physically showing people something, perhaps throughout docs we don't use this wording but instead say "For example purposes...." or something like that

Reply:
see 05a3720953b7b3c81e7718d5131cfe6f5e78d2ba. I will keep an eye on the global comment you made. Thank you !